app/views/graders/_graders_manager.js.jsx.erb

```diff
@@ -553,7 +558,8 @@
 
       // need to filter for each section
       var filters = [];
-      filters.push({name: '-', text: '-', func: function(g) { return g; } });
+      var all = '<%= j raw I18n.t('all') %>';
```

Comment:
http://apidock.com/rails/ActionView/Helpers/JavaScriptHelper/escape_javascript and http://apidock.com/rails/ActionView/Helpers/OutputSafetyHelper/raw.

Neither apply here.